src/assets/app.config.json

```diff
@@ -1,5 +1,6 @@
 {
     "backEndHost": "HELLO_BACKEND_URL",
     "acsHost": "ACS_URL",
-    "apsHost": "APS_URL"
+    "apsHost": "APS_URL",
+    "identityServiceHost": "IDENTITY_SERVICE_URL"
```

Comment:
I think the APS and ACS properties are coming from ADF so we may have less control over the name, but they should really be URL rather than host. 

It's probably fine to keep them consistent for now but in the future we should look into correcting all them, including submitting a change request to ADF if need be.

Reply:
I agree with Ray's thoughts on current consistency future enhancement.